src/Dom/Document.php

```diff
@@ -2108,8 +2109,13 @@ public function isCssMaxByteCountEnforced()
      *
      * @param int $maxByteCount Maximum number of bytes to limit the CSS to. A negative number disables the limit.
      */
-    public function enforceCssMaxByteCount($maxByteCount = Amp::MAX_CSS_BYTE_COUNT)
+    public function enforceCssMaxByteCount($maxByteCount = null)
     {
+        if ($maxByteCount === null) {
+            // No need to instantiate the spec here, we can just directly reference the needed constant.
+            $maxByteCount = AmpTransformed::SPEC[SpecRule::MAX_BYTES];
```

Comment:
This is currently being set to `112500` not the expected `75000`. So this change is needed:

```suggestion
            $maxByteCount = AmpNoTransformed::SPEC[SpecRule::MAX_BYTES];
```

Reply:
Yes, I used the `AmpTransformed` one because my assumption was that we are using this for the Optimizer output (which should always be transformed).

However, I realize now that this particular default is set in the `Dom\Document` class, which is not limited to the Optimizer.

So I guess using the `AmpNoTransformed` one and then adapting the limit while running the Optimizer is the way to go.